src/components/transaction-finish.tsx

```diff
@@ -137,12 +137,12 @@ export const TransactionFinish = observer(
           <Text
             variant={TextVariant.t14}
             color={Color.textBase2}
-            i18n={I18N.transactionConfirmationHAQQ}
+            children={app.provider.name}
```

Comment:
Do not pass children as props. Instead, nest children between the opening and closing tags.sonarlint(typescript:S6748)